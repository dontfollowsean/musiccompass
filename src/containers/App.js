// @flow

import React from 'react';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import Header from '../components/Header';
import Landing from './Landing';
import './App.css';

const App = () => (
	<MuiThemeProvider>
		<div className="App">
			<Header />
			<Landing />
		</div>
	</MuiThemeProvider>
);

export default App;
