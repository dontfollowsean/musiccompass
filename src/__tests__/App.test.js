import React from 'react';
import { shallow } from 'enzyme';
import App from '../containers/App';

test('renders without crashing', () => {
	const component = shallow(<App />);
	expect(component).toMatchSnapshot();
});
