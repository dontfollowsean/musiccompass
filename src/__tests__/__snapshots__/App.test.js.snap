// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<MuiThemeProvider>
  <div
    className="App"
  >
    <Header />
    <Landing />
  </div>
</MuiThemeProvider>
`;
